dpnp/backend/extensions/fft/out_of_place.cpp

```diff
@@ -23,7 +23,10 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 //*****************************************************************************
 
+#if not defined(USE_ONEMKL_INTERFACES)
 #include <oneapi/mkl/dfti.hpp>
```

Comment:
Should we include "oneapi/mkl/dft.hpp" header here instead in case when USE_ONEMKL_INTERFACES is defined?
Since we are using at line#43, it looks we included that somewhere explicitly.

Reply:
`#include <oneapi/mkl.hpp>` Included in `out_of_place.hpp` file.  There is no need to include `oneapi/mkl/dft.hpp` directly.

Reply:
Is there any difference in required includes when `USE_ONEMKL_INTERFACES` defined or not?
Should we align the includes in both cases?
In general, I think it's good to have explicit includes if the functionality is used by that file.